D3D11Engine/GothicAPI.cpp

```diff
@@ -653,12 +653,10 @@ void GothicAPI::LoadRendererWorldSettings( GothicRendererSettings& s ) {
         GetPrivateProfileIntA( "Atmoshpere", "FogColorModB", (int)(s.FogColorMod.z * 255.0f), ini.c_str() )
     );
 
-    if ( !GMPModeActive ) {
```

Comment:
Owners and players of GMP servers wanted to have the limit.

This should also have been enforced in the F11 menu. Had to hand merge a few PRs, maybe one of them deleted the check in F11 and when starting the game.

Reply:
I see no benefit in introducing this limit, because you can't see other players nevertheless if the GMP doesn't stream them. And without the renderer you could change the limit with the Systempack anyway.

Reply:
Fair enough. I don't know anything about GMP, just what the people tell me. If GMP really streams people and items only if they are in the "visible range" (defined by GMP?) and that works with Systempack scaling etc. then that's fine.

Reply:
Well, I'm the developer of the GMP :D The distance to which other players are streamed can be set by the servers, but items are indeed always visible. But I don't think that's a problem.